guides/doc-Release_Notes/master.adoc

```diff
@@ -10,6 +10,10 @@ ifdef::katello[]
 
 include::topics/foreman.adoc[leveloffset=+1]
 include::topics/katello.adoc[leveloffset=+1]
+endif::[]
+
+// Start inserting specific x.y.z releases here
+ifdef::katello[]
 include::topics/katello-4.6.0.adoc[leveloffset=+1]
```

Comment:
Maybe use an attribute for Katello version?

Reply:
Are you suggesting this:
```suggestion
include::topics/katello-{KatelloVersion}.0.adoc[leveloffset=+1]
```

I'm not sure it helps that much. If you go that route, you could also consider naming the file `topic/katello-0.adoc` (and `topic/katello-1.adoc` for Katello 4.6.1). However, then we'd need to change it everywhere IMHO.

I'd prefer to merge this now and if you feel it's something we should do then we open an issue to refactor all branches.

Reply:
OK, good point. Disregard this suggestion then.